### DISCUSSION TITLE
One string in HTML keeps being reset?

### DISCUSSION THREAD

**Discussion by nclm** [WeblateOrg/weblate#12498] [Q&A]
Hello,

I have an HTML page translated with Weblate that has this line:

`<label title="Dark/Light" for="color-scheme-btn" class="color-scheme-btn-label">Dark/Light</label>`

Like the rest of the content on the page, the automatic HTML parser detects it properly and provides the string Dark/Light for translation.

However, every single time something else is changed on the page, “Dark/Light” is somehow re-parsed as a new string and appears on Weblate as an untranslated string. We also loose all the existing translations for that string, so translators have to fill it again.

How to make it so that Weblate recognise that it’s the same string and do not reset it every single change? All the other unchanged content on the page is properly being recognised, it’s only this label that bugs each time.

Thanks!

Bonus and related question:

That line is actually turned into two identical strings (both mistakenly reset at every page change): “Dark/Light” (the title) and “Dark/Light” (the HTML content of the tag). Is there a way to tell Weblate: when the string is perfectly identical, just consider it as one string?

**Comment by nijel** [comment]
Is this on the current Weblate version, or on some older release? This really looks like behavior before 1c94fda0068fabb7df6fcef4525f94870dff4b20 or even abedcbacf714353bc2032a3bb086d90b1735dbc6.

**Reply by nijel** [reply]
Ah, the `<label>` tag is not being extracted right now, so you only translate the title. https://github.com/translate/translate/pull/5353 should address this.

**Comment by nclm** [comment]
This is the Weblate installed on https://translate.codeberg.org, which is now 5.7.2 according to the footer
